nodemon.json

```diff
@@ -0,0 +1,9 @@
+{
+  "watch": ["api/", "api-lib/", "scripts/"],
```

Comment:
Nice!

Reply:
`src` needs to be in this list too:

```
# this is a lower bound because it excludes multi-line imports
❯ ack -l "import.*src" api api-lib
api/hasura/actions/_handlers/createUserWithToken.ts
api/hasura/actions/_handlers/allocationCsv.ts
api/hasura/actions/_handlers/createVault.ts
api/hasura/actions/_handlers/updateContribution.test.ts
api/hasura/actions/_handlers/logoutUser.ts
api/hasura/actions/_handlers/deleteContribution.test.ts
api/hasura/cron/recoverTransactions.ts
api-lib/signature.ts
api-lib/HttpError.ts
api-lib/images.ts
```